java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java

```diff
@@ -162,11 +169,11 @@ public Builder withAppendLog(boolean appendLog) {
     /**
      * Configures the driver server verbosity.
      *
-     * @param verbose True for verbose output, false otherwise.
+     * @param logLevel {@link ChromeDriverLogLevel} for desired log level output.
      * @return A self reference.
      */
-    public Builder withVerbose(boolean verbose) {
-      this.verbose = verbose;
```

Comment:
`withVerbose` is a general API that all DriverServiceBuilders expose. Best to have this delegate down to `withLogLevel` and have it set to `DEBUG` or `ALL`. That way, people trying to do generalised set up get what they expect, and power users can do something more fine-grained.